### PR TITLE
[16.0][MIG] base_fontawesome

### PR DESCRIPTION
**Pull request by legalsylvain**
Port ``base_fontawesome`` from 15.0 to 16.0
- remove useless file
- adapt asset replacement to new location

- DONT change the fontawesome. (still the 5.15.4, not the 6.2.0)
(I don't need it, but feel free to make a PR against mine, or a second PR once this one is merged).

### CODE REVIEW

**Comment by legalsylvain** [comment]
/ocabot migration base_fontawesome

CC : @simahawk, @etobella , @olgamarcocb

File: base_fontawesome/__manifest__.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 {
     "name": "Base Fontawesome",
     "summary": """Up to date Fontawesome resources.""",
-    "version": "15.0.5.15.4",
+    "version": "16.0.1.0.1",
```
Comment (etobella) [inline base_fontawesome/__manifest__.py]: We usually tried to keep the version of font awesome on the version of the module, but it is ok for me to change it

**Review by etobella — APPROVED** [review]

File: base_fontawesome/__manifest__.py (hunk 1 above)
Comment (simahawk) [reply base_fontawesome/__manifest__.py]: I'd stick to that rule

**Review by simahawk — APPROVED** [review]

**Review by nimarosa — APPROVED** [review]

**Comment by StefanRijnhart** [comment]
Setting to 'Needs fixing' for the version number alignment.

File: base_fontawesome/__manifest__.py (hunk 1 above)
Comment (legalsylvain) [reply base_fontawesome/__manifest__.py]: Fixed. I didn't know that rule. Thanks for your review.

(could be merged I guess now).

**Review by etobella — APPROVED** [review]

**Pull request merged**